spaces/S000174/properties/P000163.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000174
+property: P000163
+value: true
+refs:
+  - doi: 10.4064/FM-97-2-53-55
+    name: "A perfectly normal locally metrizable non-paracompact space"
+---
+
+By construction, the underlying set has a metrizable topology with a base of cardinality $\aleph_1$. Thus, $|X| \leq \aleph_1^\omega \leq (2^\omega)^\omega = 2^\omega = \mathfrak{c}$.

```

Comment:
It'd be nice to have a reference for $|X|\leq|w(X)|^\omega$ for metrizable spaces; it sounds right but it's not something I would spout off-hand if asked.

Reply:
@StevenClontz I'm not sure whether the general case has been explicitly written down anywhere, but in this specific case it is effectively already mentioned in [T404](https://topology.pi-base.org/theorems/T404), where it is noted that for a sequential, US space $X$, if it has density character at most continuum, then its cardinality is at most continuum as well. The same argument cited in T404 can be generalized to show that $|X| \leq \chi(X)^\omega$ when $X$ is sequential and US, where $\chi(X)$ is the density character of $X$. And, of course, $\chi(X) \leq w(X)$.

Reply:
@StevenClontz I'll just cite the MSE answer referenced in T404 for this.

Reply:
Oh, I should have remembered T404 since I participated in those Math SE comments. 😅

If it's relevant for your update, note that $d(X)$ is more common for density, while $\chi(X)$ is used for the supremum of local weights. https://en.wikipedia.org/wiki/Cardinal_function

I think this is really telling us we need to add $d(X)\leq\mathfrak c$ as a property, and update T404. That can be done after this.